Routing/SegmentVariant/Root.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Routing.SegmentVariant
+{
+    [Equals]
+    internal class Root : ISegmentVariant
+    {
+
```

Comment:
Remove this blank line

Reply:
https://github.com/messerli-informatik-ag/routing/pull/6/commits/ede78780d40dc6aa81b6837a3536a9538b64f89d